scripts/database/create_tables.sql

```diff
@@ -74,3 +74,69 @@ BEGIN
       PREFERRED_LANGUAGE                NVARCHAR (50) NULL,
 );
 END
+
+
+
+IF NOT EXISTS
+(
+    SELECT *
+    FROM INFORMATION_SCHEMA.TABLES
+    WHERE TABLE_SCHEMA = 'dbo'
+    AND TABLE_NAME = 'PARTICIPANT_SCREENING_PROFILE'
+)
+BEGIN
+    CREATE TABLE PARTICIPANT_SCREENING_PROFILE
+    (
+      NHS_NUMBER                             VARCHAR(50) NULL,
+      SCREENING_NAME                         VARCHAR(50) NULL,
+      PRIMARY_CARE_PROVIDER                  VARCHAR(50) NULL,
+      PREFERRED_LANGUAGE                     VARCHAR(50) NULL,
+      REASON_FOR_REMOVAL                     VARCHAR(50) NULL,
+      REASON_FOR_REMOVAL_DT                  VARCHAR(50) NULL,
+      NEXT_TEST_DUE_DATE                     VARCHAR(50) NULL,
+      NEXT_TEST_DUE_DATE_CALCULATION_METHOD  VARCHAR(50) NULL,
+      PARTICIPANT_SCREENING_STAT             VARCHAR(50) NULL,
```

Comment:
Should be STATUS rather than STAT

Reply:
sorted :)

Reply:
Do you still need to push the change?

Reply:
oops pushing now :)

Reply:
Remember to re-scaffold the entity model and db context

Reply:
ah will do :)

Reply:
ok re-scaffold complete